docs/docs/examples/full-pipelines/multi-repo/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Multi-Repository Code Locations
+description: Learn how to set up multiple code locations in Dagster+ with cross-repository asset dependencies
+last_update:
+  author: Dennis Hume
+sidebar_custom_props:
+  logo: images/dagster-primary-mark.svg
+tags: [code-example]
+canonicalUrl: '/examples/full-pipelines/multi-repo'
+slug: '/examples/full-pipelines/multi-repo'
+---
+
+# Multi-Repository Code Locations
```

Comment:
I think we should have a "see the code" link right at the top of any full pipeline example (I know I've tried to find this for e.g. the Bluesky one and putting it right at the top would be super helpful